packages/terra-site/package.json

```diff
@@ -22,7 +22,7 @@
     "test:spec": "echo \"Warning: no test written\" && exit 0"
   },
   "dependencies": {
-    "moment": "^2.17.1",
+    "moment": "^2.18.1",
```

Comment:
https://github.com/Hacker0x01/react-datepicker/blob/master/package.json#L94
I would recommend we match the same version of moment that react-datepicker has.

```
"moment": "^2.17.1",
```